### PR TITLE
Fix mpl pyqt4 backend selection when running sliceviewer stand-alone

### PR DESCRIPTION
**Pull request by rosswhitfield**
Starting the sliceviewer outside the workbench using pypt4 would fail when using old versions of matplotlib, it should now import the correct backend.

**To test:**
See example in issue #25619

*This does not require release notes* because sliceviewer hasn't been added to the release notes yet.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by peterfpeterson — DISMISSED** [review]
Exactly the right fix. This works on RHEL7.

**Comment by gvardany** [comment]
Is this closing the issue then?

File: qt/python/mantidqt/widgets/sliceviewer/view.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from qtpy.QtWidgets import QWidget, QVBoxLayout
 from qtpy.QtCore import Qt
 from matplotlib.figure import Figure
-from matplotlib.backends.backend_qt5agg import FigureCanvas, NavigationToolbar2QT as NavigationToolbar
+from MPLwidgets import FigureCanvas, NavigationToolbar2QT as NavigationToolbar
```
Comment (martyngigg) [inline qt/python/mantidqt/widgets/sliceviewer/view.py]: Any chance we could move the functionality in `MPLwidgest` to `mantidqt` as a util to import the matplotlib backend? It'd be nice if `mantidqt` could be standalone.

Comment (rosswhitfield) [reply qt/python/mantidqt/widgets/sliceviewer/view.py]: I did think using something from `scripts` wasn't the best thing to do. Do you think I could just move https://github.com/mantidproject/mantid/blob/master/scripts/MPLwidgets.py and https://github.com/mantidproject/mantid/blob/master/scripts/gui_helper.py into `qt/python/mantidqt` and update the imports?

Comment (martyngigg) [reply qt/python/mantidqt/widgets/sliceviewer/view.py]: I think that would be fine.

**Review by VickieLynch — COMMENTED** [review]
Thanks for fixing this. Works for me now from qt4

**Comment by rosswhitfield** [comment]
I moved` MPLwidgets.py`/`gui_helper.py` to `mantidqt`, so this probably needs to be tested again to check that I didn't break any interfaces in MantidPlot or MantidWorkbench.

**Review by martyngigg — APPROVED** [review]
Works for me now with both pyqt and pyqt5

**Pull request merged**